lib/main.dart

```diff
@@ -216,7 +216,12 @@ class _MyHomePageState extends State<MyHomePage> {
   }
 }
 
+// Maybe add input validation here, it is true that when you add a task with empty fields you cannot do it
```

Comment:
Error message would be more user friendly and allow him to understand quicker what is wrong.